share/completions/zef.fish

```diff
@@ -0,0 +1,94 @@
+complete -c zef -f
+complete -c zef -n __fish_is_first_token -s h -l help -f -d 'Display help text'
+complete -c zef -n __fish_is_first_token -l version -f -d 'Detailed version information'
+complete -c zef -n 'not __fish_is_first_token' -l config-path -d 'Load a specific Zef config file'
+complete -c zef -n '__fish_seen_argument -l config-path' -x -k -a '(__fish_complete_suffix)'
```

Comment:
Which suffix?

If you just want file completions, use `-F`/`--force-files`.

Reply:
I tried that first, but it doesn't seem to work. Maybe because of `complete -c zef -f`. I tried different different combinations, and this was the only thing that worked for me.

Reply:
Hmm, I tried it `-F` again on `--config-path` and it seemed to work, this time! Will try it on other similar options.

Reply:
OK, it worked on all, but one:
```fish
complete -c zef -n '__fish_seen_subcommand_from install test build' -k -a '(__fish_complete_suffix)'
```
I don't know if there is another way.

Reply:
Does

```fish
complete -c zef -n '__fish_seen_subcommand_from install test build' -F
```

not work?

Reply:
No. At least, not for me.